src/components/Table/GridTable.tsx

```diff
@@ -1027,6 +1038,11 @@ function isContentAndSettings(content: ReactNode | GridCellContent): content is
   return typeof content === "object" && !!content && "content" in content;
 }
 
+const emptyValues = ["", null, undefined] as any[];
```

Comment:
Nice! I wonder how many cells this is going to actually change... :thinking: Will be interesting to see.